pkg/ip/ip.go

```diff
@@ -852,6 +852,16 @@ func ListContainsIP(ipList []net.IP, ip net.IP) bool {
 	return false
 }
 
+// ListContainsAddr returns whether a list of IPs contains a given IP.
+func ListContainsAddr(ipList []netip.Addr, ip netip.Addr) bool {
```

Comment:
Why not use [`slices.Contains`](https://pkg.go.dev/slices#Contains)?

Reply:
Or [`golang.org/x/exp/slices.Contains`](https://pkg.go.dev/golang.org/x/exp/slices#Contains) in case this needs to be used with Go ≤ 1.21.

Reply:
good shout, I just copy pasted the `ListContainsIP` implementation, but given that with `netip.Addr` we use the compare operator, I'll update it